src/server_manager/web_app/ui_components/outline-server-settings.js

```diff
@@ -356,5 +359,20 @@ Polymer({
     const port = Number(value);
     const valid = !Number.isNaN(port) && port >= 1 && port <= 65535 && Number.isInteger(port);
     return valid ? '' : this.localize('error-keys-port-bad-input');
+  },
+
+  _getInternationalizedUnit(bytesAmount, language) {
+    return formatBytesParts(bytesAmount, language).unit;
+  },
+
+  _getTranslatedDate(language, date) {
```

Comment:
A name like `formatDate` would be more appropriate. There's no translation (from one language to another) going on.

Reply:
Done in 435c5a5a56dab1230b3f11762438144ba3f5d7cf